### DISCUSSION TITLE
Filter is in

### DISCUSSION THREAD

**Discussion by uzemelteteshj** [Budibase/budibase#10443] [Help] [answered]
Hi,

I want to use text field filter. 
in DB text ,,STRELNYK PAVLO" if i want to searching ,,stre" (Start with) is work but if i searching ,,PAV" i cant se results.

![image](https://user-images.githubusercontent.com/113426887/235123202-5c884b9e-de31-4b4f-bec1-a75ebb6ccad6.png)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @uzemelteteshj,

I'm assuming you are using `Starts with`, this will only matches results if the string begins with the characters of the one which you are entering. Due to your search string being in the middle its not seeing it as a valid result. There is another operator `like` which will do what you require. 

This operator is currently not available on cloud but will be in the future.

<img width="1041" alt="Screenshot 2023-04-28 at 12 53 49" src="https://user-images.githubusercontent.com/126772285/235140750-844cc53b-7296-42ef-aa23-ba8c25fd0029.png">

I hope this helps!

**Reply by uzemelteteshj** [reply]
I use in docker onpromise and there is no other option

**Reply by ConorWebb96** [reply]
Hey @uzemelteteshj,

This `like` operator is also currently not available for docker versions. This will be available next week whenever the PR here: https://github.com/Budibase/budibase/pull/10366 is merged in and deploy.

I hope this makes sense.

**Reply by uzemelteteshj** [reply]
Tahnk you! :)

**Comment by sghriyadh** [comment]
Is like filter Available now for docker self hosted version?

**Reply by uzemelteteshj** [reply]
yes!
